deploy/dashboards/vertx.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: "monitoring.kiali.io/v1alpha1"
+kind: MonitoringDashboard
+metadata:
+  name: vertx
```

Comment:
Missing labels for app and version. This will not be removed with selector=app=kiali without it.

Reply:
At the moment it's not meant to be tied to Kiali - not deployed with Kiali and not removed with it.
It's currently here as an example, at least until the whole epic is finished.